tests/snuba/api/endpoints/test_organization_group_index_stats.py

```diff
@@ -63,6 +64,33 @@ def test_simple(self):
         assert "lifetime" in response_data[0]
         assert "filtered" in response_data[0]
 
+    def test_issue_platform_issue(self):
+        event = self.store_event(
+            data={"timestamp": iso_format(before_now(seconds=1)), "fingerprint": ["group-a"]},
+            project_id=self.project.id,
+        )
```

Comment:
Hmm, I don't think this will create the right `GroupType` when we store it like this. If we assert the type, it looks like an error issue?